.coveragerc

```diff
@@ -59,7 +59,7 @@ fail_under=0
 ignore_errors = False
 include=
 omit=
-    */isodatetime/tests*
+    */isodatetime/test*
```

Comment:
Opsie! My bad

Reply:
Well you were not wrong there as we only had the single `isodatetime.tests` module.

Reply:
Oh! I see! Thought I misspelled the test directory. Need coffee ☕️